### DISCUSSION TITLE
Updating a title isn't firing any ajax to save this change to a table

### DISCUSSION THREAD

**Discussion by adam-jones-net** [area17/twill#1120] [Ideas]
I've multiple sites in development with totally different configurations in progress, yet via one of them by total chance I just discovered that the feature to edit the title of a record isn't working.   Checking my other twill sites the problem is also present in all the others.

The DOM updates with the new title and when you click the UPDATE button the popup closes. However no network communication has occurred and therefore no request to save this change to the related slugs table has been made.

This must be something really basic.  I assumed this kind of functionality was built in but is that not the case?  Do I need to add something for this ?

**Comment by yanhao-li** [comment]
@arkid 
I think that's true, when you click the update, it should only persist it in the vuex store, you have to click the `Update` button on the main form again to save it as a whole.
Does that bother you? would love to hear your use cases.

**Comment by daun** [comment]
I'd argue that it's confusing for users to have two buttons with the same label that don't behave in the same way. Not updating the title via ajax is probably fine if it's obvious you have to press the main `Save` button.

**Comment by yanhao-li** [comment]
@arkid @philippdaun 

You guys have persuaded me to change my mind, now I agree it's definitely a place that could be improved, and there is no harm to send an extra ajax even if people are going to click the `Update` on the main form button later.

I think this would be a great first issue if anyone would love to contribute, I will add tags to it.

Thank you both!
